code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -901,7 +901,12 @@
 		M.emote("nya")
 	if(prob(20))
 		to_chat(M, "<span class = 'notice'>[pick("Headpats feel nice.", "Backrubs would be nice.", "Mew")]</span>")
-	M.adjustArousalLoss(5)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/list/adjusted = H.adjust_arousal(5,aphro = TRUE)
+		for(var/g in adjusted)
+			var/obj/item/organ/genital/G = g
+			to_chat(M, "<span class='userlove'>You feel like playing with your [G.name]!</span>")
```

Comment:
This is gonna be spammy. Make this a probability, and pick one organ, not every one.

Reply:
Re-read adjust arousal.

Reply:
Umh, yea, my bad.